### PR TITLE
NMA-5949: Fix content colors for SatsEmptyState component

### PR DESCRIPTION
**Pull request by sindrenm**
## Before

image

## After

image

### CODE REVIEW

**Review by cecilieopperudsats — APPROVED** [review]
I am content with this.

File: sats-dna/src/main/kotlin/com/sats/dna/components/SatsEmptyState.kt [hunk 1]
```diff
@@ -120,6 +118,23 @@ private fun SatsEmptyStatePreview() {
...
 }
 
+@LightDarkPreview
+@Composable
+private fun SatsEmptyStateM2Preview() {
+    SatsTheme {
+        SatsSurface(color = SatsTheme.colors2.backgrounds.primary.bg.default, useMaterial3 = false) {
+            SatsEmptyState(
+                icon = SatsTheme.icons.barbell,
+                title = "You don't have friends",
```
Comment (oas004) [inline sats-dna/src/main/kotlin/com/sats/dna/components/SatsEmptyState.kt:128]: You have me @sindrenm ❤️

Comment (sindrenm) [reply sats-dna/src/main/kotlin/com/sats/dna/components/SatsEmptyState.kt:128]: Aww! ❤️

**Comment by SatseHavard** [comment]
Stakkars ent

**Comment by sindrenm** [comment]
Stakkars nent!

**Pull request merged**